### PR TITLE
crossPaths back to default 

### PR DESCRIPTION
**Pull request by johanandren**
Refs #27854

+publish now works locally at least

### CODE REVIEW

**Comment by johanandren** [comment]
Also looked through the usages of `target`, looks like all usages are ok (we already are using `crossTarget` in a few places)

File: project/Publish.scala [hunk 1]
```diff
@@ -16,14 +16,15 @@ object Publish extends AutoPlugin {
   override def trigger = allRequirements
 
   override lazy val projectSettings = Seq(
-    crossPaths := false,
```
Comment (patriknw) [inline project/Publish.scala]: did you try a release dry-run?
does the "release" directory match the expected location for rsync and such (in release sh script)

for example we have `crossTarget.value / "release"`

Comment (johanandren) [reply project/Publish.scala]: Yes, that's right, good catch. I was thinking about it the wrong way. Now fixed and double checked with a release dry-run.

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
let's merge this before end of today

**Comment by patriknw** [comment]
merging this to have it in for the nightly

**Comment by pvlugter** [comment]
👍

**Pull request merged**